src/Xamarin.Android.Build.Tasks/Tasks/CopyAndConvertResources.cs

```diff
@@ -85,11 +98,32 @@ public override bool Execute ()
 				var dstmodifiedDate = File.Exists (destfilename) ? File.GetLastWriteTimeUtc (destfilename) : DateTime.MinValue;
 				var tmpdest = Path.GetTempFileName ();
 				var res = Path.Combine (Path.GetDirectoryName (filename), "..");
-				MonoAndroidHelper.CopyIfChanged (filename, tmpdest);
-				MonoAndroidHelper.SetWriteable (tmpdest);
+				MonoAndroidHelper.CopyIfChanged (filename, destfilename);
+				MonoAndroidHelper.SetWriteable (destfilename);
```

Comment:
the `destfilename` is no longer required, `UpdateXmlResource` can act directly on `filename`